compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -211,13 +213,35 @@ object Completion {
         // import a.C
         def isSameSymbolImportedDouble =  denotss.forall(_.denots == first.denots)
 
+        def isScalaPackage(scopedDenots: ScopedDenotations) =
+          scopedDenots.denots.exists(_.info.typeSymbol.owner == defn.ScalaPackageClass)
+
+        def isJavaLangPackage(scopedDenots: ScopedDenotations) =
+          scopedDenots.denots.exists(_.info.typeSymbol.owner == defn.JavaLangPackageClass)
+
+        // For example
+        // import java.lang.annotation
+        //    is shadowed by
+        // import scala.annotation
+        def isJavaLangAndScala =  denotss match
+          case List(first, second) =>
+            isScalaPackage(first) && isJavaLangPackage(second) ||
```

Comment:
We shouldn't assume that in case of a conflict there would be only 2 colliding items.
Consider this code (which should be added as a test case)
```scala
import java.lang.annotation; import annot${m1}
```
Here the completion should be `("annotation", Module, "java.lang.annotation")` but currently no completion is returned.

Reply:
Ach damn, so maybe:
`denotss.forall(d => isScalaPackage(d) || isJavaLangPackage(d))` 
would work?

Reply:
Should be good now!